public/game-master/styles/game-master.styl

```diff
@@ -0,0 +1,63 @@
+.game-master-screen
+	.user-panel
+		width: 100%
+
+.game-master-screen-container
+	width: 96%
```

Comment:
I was thinking about it... And then decided that I don't like too deep nesting. Because of names there should not be any collisions. Anyway if you would like - it's not a problem :)